students/hyska_monika/lesson_02_flow_control/TheSecondMaximum.py

```diff
@@ -0,0 +1,30 @@
+# Program return the second from the biggest number
+# N = [1, 12, 9, 7, 8, 10]
+# N = [1, 7, 9, 0]
+
+n = int((input("put haw many elements want check:")))
+N = []
+i = 0
+
+while i < n:
+    print("Put", i + 1, "element:")
+    l1 = int(input())
+    N.append(l1)
+    i += 1
+
+i = 0
+sm = 0
+max1 = 0
+
+while i < len(N):
```

Comment:
Here again they want a zero terminated input, not asking about number of elements.